bids/layout/layout.py

```diff
@@ -278,10 +276,10 @@ def _get_layouts_in_scope(self, scope):
         # Determine which BIDSLayouts to search
         layouts = []
         scope = listify(scope)
-        if scope is None or scope == 'all' or 'raw' in scope:
+        if scope is None or 'all' in scope or 'raw' in scope:
```

Comment:
Should we still be checking `scope is None` now that `None` isn't equivalent to `'all'`? I haven't checked all the paths that we can get here by...